language/move-lang/tests/move_check/parser/function_type_trailing_comma.move

```diff
@@ -0,0 +1,3 @@
+module M {
+    fn<T,>() { } // Test a trailing comma in the type parameters
```

Comment:
Oof, Didn't expect this to work. We should consider not allowing it

Reply:
As discussed in https://github.com/libra/libra/pull/1973, we will continue to allow this.